dev/sg/internal/rfc/rfc_create.go

```diff
@@ -0,0 +1,313 @@
+			if !foundTitle {
+				// First paragraph is the title
+				content := elem.Paragraph.Elements[0].TextRun.Content
+				matches := rfcDocRegex.FindStringSubmatch(content)
+				if len(matches) != 5 {
+					return errors.Errorf("Document format mismatch")
+				}
+				rfcSize := int64(len(matches[1]))
+				numberSize := int64(len(matches[2]))
+				titleSize := int64(len(matches[4]))
+
+				nextRfcIdStr := strconv.Itoa(nextRfcId)
+				change = append(change, []*docs.Request{
+					// Replace the title
+					{
+						DeleteContentRange: &docs.DeleteContentRangeRequest{
+							Range: &docs.Range{
+								StartIndex: elem.EndIndex - titleSize - 1,
+								EndIndex:   elem.EndIndex - 1,
+							},
+						},
+					},
+					{
+						InsertText: &docs.InsertTextRequest{
+							Location: &docs.Location{Index: elem.EndIndex - titleSize - 1},
+							Text:     title,
+						},
+					},
+				}...)
+
+				// Replace the number
+				change = append(change, []*docs.Request{
+					{
+						DeleteContentRange: &docs.DeleteContentRangeRequest{
+							Range: &docs.Range{
+								StartIndex: elem.StartIndex + rfcSize,
+								EndIndex:   elem.StartIndex + rfcSize + numberSize,
+							},
+						},
+					},
+					{
+						InsertText: &docs.InsertTextRequest{
+							Location: &docs.Location{Index: elem.StartIndex + 4},
+							Text:     nextRfcIdStr,
+						},
+					},
+				}...)
+
+				if driveSpec == PrivateDrive {
+					// Add "PRIVATE" to the title
+					change = append(change, &docs.Request{
+						InsertText: &docs.InsertTextRequest{
+							Location: &docs.Location{
+								Index: elem.StartIndex + rfcSize + rfcSize,
+							},
+							Text: "PRIVATE ",
+						},
+					})
+				}
+
+				foundTitle = true
+			}
+		}
+
+		if elem.Table != nil {
+			// First table is the reminder
+			if !foundReminder {
+				if len(elem.Table.TableRows) != 1 ||
+					len(elem.Table.TableRows[0].TableCells) != 1 ||
+					len(elem.Table.TableRows[0].TableCells[0].Content) != 1 ||
+					len(elem.Table.TableRows[0].TableCells[0].Content[0].Paragraph.Elements) == 0 {
+					return errors.Errorf("Reminder table not found")
+				}
+
+				content := elem.Table.TableRows[0].TableCells[0].Content[0].
+					Paragraph.Elements[0].TextRun.Content
+				if strings.Contains(content, "Rename this RFC in this format") {
+					// Remove the reminder, as we are doing for the user
+					change = append([]*docs.Request{{
+						DeleteContentRange: &docs.DeleteContentRangeRequest{
+							Range: &docs.Range{
+								StartIndex: elem.StartIndex,
+								EndIndex:   elem.EndIndex,
+							},
+						},
+					}}, change...)
+
+					foundReminder = true
+				}
+			}
+		}
+	}
+
+	if _, err := docService.Documents.BatchUpdate(newFile.Id, &docs.BatchUpdateDocumentRequest{
+		Requests: change,
+	}).Do(); err != nil {
+		return errors.Wrap(err, "Cannot update RFC title")
+	}
+
+	return nil
+}
+
+func createRFCs(ctx context.Context, title string, rfcTemplate RfcTemplate, driveSpec DriveSpec,
```

Comment:
Since we're only creating one RFC
```suggestion
func createRFC(ctx context.Context, title string, rfcTemplate RfcTemplate, driveSpec DriveSpec,
```